build/lily/notifier.env

```diff
@@ -5,16 +5,19 @@ LILY_REPO=/var/lib/lily
 #LILY_CONFIG=/var/lib/lily/config.toml
 
 # Postgres URL may be overridden
-#LILY_STORAGE_POSTGRESQL_DB_URL=postgres://user:pass@host:port/database?options
```

Comment:
I think we'll want to remove the quotes from here and worker.env, else the container see:`"\"postgres://postgres@timescaledb:5432/postgres\""`